apps/web/src/components/transactions/QueuedTxSimulation/index.tsx

```diff
@@ -108,12 +109,16 @@ const InlineTxSimulation = ({ transaction }: { transaction: TransactionDetails }
       <ExternalLink href={simulationLink}>
         <Stack direction="row" alignItems="center" gap={0.5}>
           <SvgIcon
-            color={status.isSuccess ? 'success' : 'error'}
-            component={status.isSuccess ? CheckIcon : CloseIcon}
+            color={status.isCallTraceError ? 'warning' : status.isSuccess ? 'success' : 'error'}
+            component={status.isCallTraceError ? WarningIcon : status.isSuccess ? CheckIcon : CloseIcon}
             inheritViewBox
             sx={{ height: '16px' }}
           />
-          {status.isSuccess ? 'Simulation successful' : 'Simulation failed'}
+          {status.isCallTraceError
+            ? 'Can execute (with warnings)'
+            : status.isSuccess
+              ? 'Simulation successful'
+              : 'Simulation failed'}
```

Comment:
Please don't do nested ternaries like this. Instead move it to a function tha receives isCallTraceErrro and isSuccess and use normal returns. e.g.
function ExecutionStatus(isCallTraceError, isSuccess) {
if(isCallTraceerror) {
return 'Can execute (with warnings)'

if (isSuccess) {
return 'Simulation successful'
}

return 'Simulation failed'
}
}

by abstracting it like this you can even easily write a small unit test.